fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1774,28 +1773,6 @@ public void testResponseContentTypes() throws Exception {
         }
     }
 
-    /**
-     * Given a directory at: test-FileSystem1/ /ds1 /ds2 /TestSubdir/ and a projection of test-objects as
-     * fedora:/files, then I should be able to retrieve an object from fedora:/files/FileSystem1 that lists a child
-     * object at fedora:/files/FileSystem1/TestSubdir and lists datastreams ds and ds2
-     *
-     * @throws IOException thrown during this function
-     */
-    @Test
-    public void testGetProjectedNode() throws IOException {
```

Comment:
I lost those test methods in: https://github.com/fcrepo4-exts/fcrepo-connector-file/blob/master/src/test/java/org/fcrepo/integration/connector/file/FileConnectorIT.java#L128

That's great.